packages/neon-core/src/wallet/Account.ts

```diff
@@ -88,6 +88,20 @@ export class Account implements NeonObject<AccountJSON> {
     });
   }
 
+  public static validateKey(str: string): string | void {
+    switch (true) {
+      case isPrivateKey(str): return "PrivateKey";
+      case isPublicKey(str, false): return "PublicKey_Decoded";

```

Comment:
Prefer to follow the current name [PublicKeyUnencoded ](https://github.com/CityOfZion/neon-js/blob/ffbf31e45c9b69ffc02e9cb7b0aad746b67400d0/packages/neon-core/src/wallet/core.ts#L49C20-L49C37)

Similarly for `PublicKeyEncoded`.